### ISSUE TITLE
invalid syntax - create_or_update_nb_obj

### ISSUE THREAD

**Issue by blacksocket** [drygdryg/netbox-device-autodiscovery#19]
Hello,

I have installed/configured the tool and getting the following error when launching the script "python3 run.py":
Python 3.7.3
root@.../netbox-device-autodiscovery# python3 run.py
  File "run.py", line 231
    if obj_id := create_or_update_nb_obj(obj_type, obj):
               ^
SyntaxError: invalid syntax

Could you please help with this issue ?

**Comment by lasag-2511** [comment]
yes i have the same issue let me know if you have found some solution
